test/absinthe/execution/subscription_test.exs

```diff
@@ -699,13 +701,11 @@ defmodule Absinthe.Execution.SubscriptionTest do
                context: %{pubsub: PubSub}
              )
 
-    assert_receive {:telemetry_event,
-                    {[:absinthe, :execute, :operation, :start], measurements, %{id: id}, _config}}
+    assert_receive {[:absinthe, :execute, :operation, :start], measurements, %{id: id}, _config}
 
     assert System.convert_time_unit(measurements[:system_time], :native, :millisecond)
 
-    assert_receive {:telemetry_event,
-                    {[:absinthe, :execute, :operation, :stop], _, %{id: ^id}, _config}}
+    assert_receive {[:absinthe, :execute, :operation, :stop], measurements, %{id: ^id}, _config}
```

Comment:
@michaelcaterisano can you fix this compiler warnings

Reply:
done in https://github.com/Frameio/absinthe/pull/4/commits/40f6cc87f5bc83a63ef979c0f485b050b0b6e696

Reply:
I am still seeing compiler warnings on this. The sha linked in Michael's comment above corresponds to another file (`lib/absinthe/phase/document/execution/resolution.ex`), which I assume also had compiler warnings at an earlier point in time.

Reply:
@GeoffreyPS The change I made was addressing a CI failure about file formatting. I just ran `mix compile` locally successfully. Where are you seeing compiler errors?

Reply:
Oh I see them here in github but not locally, will fix

Reply:
in https://github.com/Frameio/absinthe/pull/4/commits/79a0ef87d9f90e64a7f88ede1095670a67f9df8d